NEWS

```diff
@@ -11,6 +11,211 @@
 ### Features:
 ### Bugfixes:
 
+## 1.18.0-rc1 (November 21, 2024)
+### Feature
+#### UCP
+ * Improved log with explicit prompt for address v2 configuration on failure
+ * Detected device inter-node capability with a new attribute flag
+ * Enabled using CUDA staging buffers for pipeline protocols by default
+ * Added endpoint reconfiguration support for non-reused p2p scenarios
+ * Enabled non-cacheable memory domains, activated for GDR
+ * Improved message upon wireup failure
+ * Added new protocol infrastructure to store aggregated factors for all ranges
```

Comment:
imo, it's implementation detail, not really a feature

Reply:
fixed